casa/System/Aipsrc.cc

```diff
@@ -268,13 +268,13 @@ const String &Aipsrc::aipsHome() {
   return home;
 }
 
-uInt Aipsrc::registerRC(const String &keyword, Block<String> &nlst) {
+uInt Aipsrc::registerRC(const String &keyword, std::vector<String> &nlst) {
   uInt n;
-  for (n=0; n<nlst.nelements(); n++) {
+  for (n=0; n<nlst.size(); n++) {
     if (nlst[n] == keyword) break;
   }
   n++;
-  if (n>nlst.nelements()) {
+  if (n>nlst.size()) {
     nlst.resize(n);
   }
   nlst[n-1] = keyword;
```

Comment:
When using a `std::vector`, you can simplify this function:
- Use `if (nlst[n] == keyword) return n;` in the `for` loop.
- Replace all code after the `for` loop by `nlst.push_back(keyword); return n;`

Reply:
Good idea! Applied, but note that we have to return n+1 as it uses zero as a special value.